_states/ext_state_qvm.py

```diff
@@ -76,11 +100,11 @@ def __virtual__():
     '''
     # XXX: TEMP
     if not hasattr(qubes_utils, '__opts__'):
-        qubes_utils.__opts__ = __opts__
+        qubes_utils.__opts__ = __opts__  # NOQA
```

Comment:
@marmarek flake8 i think. But since i cleaned up my vim/synaptics-checker pipeline and only use pylint right now and we can kick it.